pavelib/js_test.py

```diff
@@ -15,6 +15,7 @@
 @task
 @needs(
     'pavelib.prereqs.install_node_prereqs',
+    'pavelib.utils.test.utils.ensure_clean_package_lock',
```

Comment:
We could also do this check as part of quality tests (eslint in particular) instead of js tests, which would allow tests to be run locally without this affecting anything.

https://github.com/edx/edx-platform/blob/31eeb16081290f5eed3453168841967f60efc6e1/pavelib/quality.py#L305-L311